app/helpers/application_helper.rb

```diff
@@ -40,7 +40,7 @@ def locate_link(location, bib, library = nil)
     if DONT_FIND_IT.include?(library)
       ''
     else
-      ' ' + link_to("[#{t('blacklight.holdings.stackmap')}]".html_safe, "#{ENV['stackmap_base']}?loc=#{location}&id=#{bib}", :target => '_blank', class: 'find-it', 'data-map-location' => location.to_s, 'data-toggle' => 'tooltip')
+      ' ' + link_to('<span class="glyphicon glyphicon-map-marker"></span> <span class="map-text">Where to Find It</span>'.html_safe, "#{ENV['stackmap_base']}?loc=#{location}&id=#{bib}", :target => '_blank', class: 'find-it', 'data-map-location' => location.to_s, 'data-toggle' => 'tooltip')
```

Comment:
It is actually already there https://github.com/pulibrary/orangelight/blob/development/config/locales/blacklight.en.yml#L32.